util/createStore.js

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
Will these files under `util/` be removed from `mup-web` or do they now live in both places?